.pre-commit-config.yaml

```
@@ -1,4 +1,23 @@
 repos:
+  - repo: local
+    hooks:
+      - id: avoid-old-website-urls
+        name: Avoid old URLs (https://elastisys.io/compliantkubernetes)
+        entry: "elastisys.io/compliantkubernetes"
+        language: pygrep
+        files: ".*[.]md$"
```

```suggestion
      - id: avoid-old-website-urls
        name: Avoid old URLs (https://elastisys.io/compliantkubernetes)
        entry: "elastisys.io/compliantkubernetes"
        language: pygrep
        types: [text]
        exclude: ^.pre-commit-config.yaml$
```
We have links in more files than just markdown.